pogobuf/pogobuf.client.js

```diff
@@ -154,12 +155,7 @@ function Client(options) {
      * @return {Promise}
      */
     this.batchCall = function() {
-        if (!self.batchRequests || !self.batchRequests.length) {
-            return Promise.resolve(false);
-        }
-
-        var p = self.callRPC(self.batchRequests);
-
+        var p = self.callRPC(self.batchRequests || []);
```

Comment:
Does `self.callRPC([])` resolve to `false`? If not, then this is technically a breaking change.